tests/conftest.py

```diff
@@ -341,3 +343,38 @@ def teardown_subscribe_to_cdn_dogfood():
 def beta_env_setup(monkeypatch):
     # This fixture is used to set FOREMAN_MAINTAIN_USE_BETA env_var
     monkeypatch.setenv('FOREMAN_MAINTAIN_USE_BETA', '1')
+
+
+@pytest.fixture(scope='function')
+def setup_epel_repository(request, ansible_module):
+    setup = ansible_module.yum(
+        name=epel_repo,
+        state='present')
+    assert setup.values()[0]["rc"] == 0
+
+    def teardown_epel_repository():
+        teardown = ansible_module.yum(
+            name='epel-release',
+            state='absent',
+            update_cache='yes')
+        assert teardown.values()[0]["rc"] == 0
+    request.addfinalizer(teardown_epel_repository)
+
+
+@pytest.fixture(scope='function')
+def setup_invalid_repository(request, ansible_module):
+    ansible_module.yum_repository(
+        name='test_repo',
+        description='repo with invalid baseurl',
+        file="test_repo",
+        baseurl=invalid_baseurl,
```

Comment:
we don't need to have invalid_baseurl constant, you can simply out any url here.

Reply:
updated PR :)